charts/open-notificaties/templates/configmap.yaml

```diff
@@ -21,4 +22,9 @@ data:
   EMAIL_USE_TLS: "True"
   {{- end }}
   IS_HTTPS: {{ if .Values.settings.isHttps }}"True"{{ else }}"False"{{ end }}
-  RABBITMQ_HOST: {{ .Values.settings.messageBroker.host }}
\ No newline at end of file
+  RABBITMQ_HOST: {{ .Values.settings.messageBroker.host }}
+  {{- if .Values.flower.enabled }}
+  {{- range $index, $index_value := .Values.flower.envVars }}
+  {{ $index }}: {{ $index_value | toString | quote }}
+  {{- end }}
+  {{- end }}
```

Comment:
I can't tell if this may or may not contain the basic auth credentials for flower, which would leak credentials in the configmap. Can you clarify that?

Reply:
There is a separate flower.secretVars available as value, which should be used for basic-auth credentials and any other Flower secrets. I've also documented this in the [README](https://github.com/open-zaak/charts/blob/04c4d0ed60a6a4317d22b46f4e6bd0eafd2ab398/charts/open-notificaties/README.md).